app/editor/src/components/button/styled/Button.tsx

```diff
@@ -48,7 +49,7 @@ export const Button = styled.button<IButtonProps>`
   background-color: ${(props) => {
     switch (props.variant) {
       case ButtonVariant.primary:
-        return '#38598a';
+        return '#003366';
```

Comment:
It'd be better to get our global theme sass variables updated and applied here.  Rather than having to find and change colors through each styled component.